2-collectors/scala-stream-collector/project/Dependencies.scala

```diff
@@ -60,6 +62,8 @@ object Dependencies {
     val yodaTime         = "joda-time"             %  "joda-time"                 % V.yodaTime
     val yodaConvert      = "org.joda"              %  "joda-convert"              % V.yodaConvert
     val kafkaClients     = "org.apache.kafka"      %  "kafka-clients"             % V.kafka
+    val googleCloud      = "com.google.cloud"      %  "google-cloud"              % V.googleCloud
+    val googleAuth       = "com.google.auth"       %  "google-auth-library-oauth2-http"              % V.googleAuth
```

Comment:
Formatting of the last row should line up with the others; if need be bump out all of the other rows to match.